sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -119,7 +131,46 @@ async def get_mint_decimals(self, mint: Pubkey) -> int:
         return self.mint_decimals_cache[str(mint)]
 
     async def assess_opportunity(self, opp: OpportunitySvm) -> BidSvm | None:
+        """
+        Method to assess an opportunity and return a bid if the opportunity is worth taking. This method always returns a bid for any valid opportunity.
+
+        Args:
+            opp: An object representing a single opportunity.
+        Returns:
+            A bid object if the opportunity is worth taking to be submitted to the Express Relay server, otherwise None.
```

Comment:
when can this return `None`?

Reply:
it doesn't return `None` because it always returns a bid. the typing here is just for illustration's sake and to provide an example of how one may write a bid assessment method (that ofc has the ability to return `None` and signal no bid to be made)